src/SprykerEco/Zed/Econda/Communication/Plugin/ProductsPlugin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Communication\Plugin;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\File\FileWriterInterface;
+use SprykerEco\Zed\Econda\Dependency\Plugin\EcondaPluginInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \SprykerEco\Zed\Econda\Business\EcondaFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Econda\EcondaConfig getConfig()
+ */
+class ProductsPlugin extends AbstractPlugin implements EcondaPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\LocaleTransfer $localeTransfer
+     * @param \Generated\Shared\Transfer\BatchResultTransfer $batchResultTransfer
+     * @param \SprykerEco\Zed\Econda\Business\Exporter\Writer\File\FileWriterInterface $dataWriter
```

Comment:
The public API should not expect something as an arugment from the deep of model